src/recon_buildblock/BinNormalisationSPECT.cxx

```diff
@@ -106,6 +105,8 @@ post_processing()
   norm_proj_data_info_ptr=ProjData::read_from_file(projdata_filename);
   max_tang=norm_proj_data_info_ptr->get_max_tangential_pos_num();
   
+  this->view_time_interval=get_exam_info_sptr()->get_time_frame_definitions().get_duration(0)/num_views;
```

Comment:
probably best to have a check here on number of time frames()==1

Reply:
something like:
if (false)
error(multiple image frame are not yet supported)

?

Reply:
also a check `f (this->get_exam_info_sptr()->get_time_frame_definitions().get_num_frames()==0) error()`